packages/reusable-blocks/src/store/actions.js

```diff
@@ -40,28 +40,43 @@ export const __experimentalConvertBlockToStatic =
 	};
 
 /**
- * Returns a generator converting one or more static blocks into a reusable block.
+ * Returns a generator converting one or more static blocks into a pattern.
  *
- * @param {string[]} clientIds The client IDs of the block to detach.
- * @param {string}   title     Reusable block title.
+ * @param {string[]} clientIds  The client IDs of the block to detach.
+ * @param {string}   title      Pattern title.
+ * @param {string}   syncType   They way block is synced, current 'fully' and 'unSynced'.
```

Comment:
Shouldn't it be `unsynced` all lowercase?

Nit: We can use enum in the type as well.

```suggestion
 * @param {'fully'|'unsynced'}   syncType   They way block is synced, current 'fully' and 'unSynced'.
```

Reply:
Fixed.